vm/src/stdlib/nt.rs

```diff
@@ -424,6 +424,39 @@ pub(crate) mod module {
         Ok(vm.ctx.new_list(drives))
     }
 
+    #[pyfunction]
+    fn listvolumes(vm: &VirtualMachine) -> PyResult<PyListRef> {
+        let mut volumes = vec![];
+        let mut buffer = [0u16; 257];
```

Comment:
257 is a very specific value. was there any comment about the magic number?

Reply:
MAX_PATH + 1 (windows max path is 256).
I will leave a comment

Reply:
If it is MAX_PATH, can we do like:
```suggestion
        let mut buffer = [0u16; MAX_PATH + 1];
```